styles/components/_layout.scss

```diff
@@ -425,3 +425,46 @@
 .posF {
   position: fixed;
 }
+.about { 
+  .brandSmall {
+    width: 50px;
+    hegiht: 50px;
+  }
+  .flexBar {
+    display: flex;
+    flex-direction: row;
+  }
+
+  .flexSpaceBetween {
+    display: flex;
+    flex-direction: row;
+    justify-content: space-between;
+  }
+  
+  #donation-qr-code {
```

Comment:
This is a generic module and stylings for a specific modal, like #donate-cta and #donate-qr-code should go into a seperate style sheet for that modal (like you were doing with _about.scss).

It's basically about determining what are generic styles, potentially applicable to many different parts of the app (which would go into some of our generic style files (mostly in the components folder) -versus- what styles are specific to a particular module (which would go into a file in the module folder).